modules/router-store/src/router_store_module.ts

```diff
@@ -302,7 +300,13 @@ export class StoreRouterConnectingModule {
   private dispatchRouterAction(type: string, payload: any): void {
```

Comment:
Will you check to see how we can improve the signature for this function? `payload: any` could be `RouterEvent` at a minimum I think.